src/keyboard.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "game_object.h"
+#include "keyboard_listener.h"
+#include "observable.h"
+#include <SDL2/SDL.h>
+#include <string>
+#include "graphics_handler.h"
+#include "events_listener.h"
+
+
+/* Keyboard
+ * Listens to events.h using the "notify_events" function. Keyboard will then look for events that are keyboard related
+ * and return characters. 
+ * Will return a single character std::string when a keypress is made
+ * Will return non-single character std::string when the following keypresses are made
+ *  [Button]            [Return Value]
+ *  Left arrow          LEFT         
+ *  Right arrow         RIGHT
+ *  Up arrow            UP
+ *  Down arrow          DOWN
+ *  Space Key           SPACE
+ *  Backspace           BKSPACE
+ *  Esc Key             ESC
+ *  Enter Key           ENTER
+ *  Forward Slash       SLASH
+ *  
+ */
+
+class Keyboard : public EventsListener, public Observable<KeyboardListener> {
```

Comment:
Keyboard is pretty general, perhaps a more precise name would be `KeyPressHandler`